docs/user_manual.md

```diff
@@ -318,6 +319,18 @@ containers, e.g. `docker run --security-opt <value...>`.
 For more details, see the [Docker
 docs](https://docs.docker.com/reference/run/#security-configuration).
 
+#### secondsToWaitBeforeKill
```

Comment:
I know the other options aren't in order, but can we put this above securityOpt?